WalletWasabi/Tor/TorProcessManager.cs

```diff
@@ -249,6 +249,16 @@ public async Task StopAsync()
 			}
 			Stop?.Dispose();
 			Stop = null;
+
+			if (TorProcess is { } && killTor)
+			{
+				Logger.LogInfo($"Killing Tor process.");
+
+				TorProcess.Kill();
```

Comment:
This may throw an exception (Tor process may be already closed (it crashed or whaever), closing, etc.) and I believe it's better to catch all exceptions here and just log it.

Reply:
Added